rebar_eunit_run

```diff
@@ -1,3 +1,3 @@
 #/bin/bash
 
-rebar3 eunit -mpop_verifier_test
\ No newline at end of file
+rebar3 eunit -mpotato_udp_test
```

Comment:
:+1:

Reply:
too lazy to type it out each time

Reply:
also too lazy to wait for all other unit tests

Reply:
:+1:  is really just for the potato